.github/config.yml

```diff
@@ -0,0 +1,11 @@
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue here at xarray! Be sure to follow the issue template!
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thank you for opening this pull request! Please check out our contributing guidelines at https://github.com/pydata/xarray/blob/main/CONTRIBUTING.md .
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Congratulations on merging your first pull request! We here at xarray are proud of you!
```

Comment:
Instructions of the bot mention that one should add as many gifs as possible, haha!

An recommendations?

Reply:
Sure!
I wasn't sure if having gifs would be okay ahaha
I'll add some now!